modules/date/date-parser-grammar.ym

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2013 BalaBit IT Ltd, Budapest, Hungary
+ * Copyright (c) 2013 Gergely Nagy <algernon@balabit.hu>
```

Comment:
You should probably change the copyright here. Remove me, and add yourself instead.